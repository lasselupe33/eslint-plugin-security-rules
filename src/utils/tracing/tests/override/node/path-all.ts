import path from "path";

const start = path.join(
  path.dirname(__filename),
  path.parse("somePath").name,
  path.normalize(__dirname),
  path.extname(__filename),
  path.delimiter,
  path.sep
);
