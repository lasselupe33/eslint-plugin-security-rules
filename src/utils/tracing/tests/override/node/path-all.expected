__filenamesomePath__dirname__filenamepathpath